ocaml/xenopsd/suspend_image_viewer/dune

```diff
@@ -1,5 +1,7 @@
 (executable
+ (public_name suspend-image-viewer)
```

Comment:
Note that the module name needs to stay with underscores because OCaml doesn't accept '-' in filenames, but the public name can indeed be anything, including '-'.